js/index.js

```diff
@@ -0,0 +1,35 @@
+document.addEventListener('DOMContentLoaded', function() { 
+    const footer = document.createElement('footer')
+// footer.textContent = 'HAUMEA 2024'
+
+
+let today = new Date();
+let thisYear =  today.getFullYear();
+let copyright = document.createElement('span');
+copyright.textContent = "Copyright ";
+
+const fullname = document.createElement('span');
+fullname.textContent = "Anna Bazileeva ";
+
+let unicode = document.createElement('span');
+unicode.textContent = " © ";
+
+footer.appendChild(copyright);
+footer.appendChild(fullname); 
+footer.appendChild(unicode)
+footer.append(` ${thisYear}`)
+
+document.body.appendChild(footer)
+})
+
+let skills = ["JavaScript", "HTML", "CSS", "GitHub"];
+let skillsSection = document.getElementById('skills');
+let skillsList = skills.getElementById('ul')
```

Comment:
The function `getElementById` is not available from an element. Also, `ul` isn't an id, it's a tag. You can use the [getElementByTagName](https://developer.mozilla.org/en-US/docs/Web/API/Element/getElementsByTagName) instead.